features/old/menu/account_menu.feature

```diff
@@ -34,14 +34,15 @@ Feature: Menu of the Account screen
     Given provider "foo.example.com" has "finance" switch denied
     When I go to the provider dashboard
      And I follow "Account"
-    Then I should see "3scale Invoices"
```

Comment:
I find it unnecessary to confirm that you can see what you are about to click in the next line 😄if you can't see it, let it fail there 😛